ntcore/src/main/native/cpp/NetworkServer.cpp

```diff
@@ -131,6 +132,7 @@ class NSImpl {
   std::shared_ptr<uv::Timer> m_savePersistentTimer;
   std::shared_ptr<uv::Async<>> m_flushLocal;
   std::shared_ptr<uv::Async<>> m_flush;
+  bool m_shutdown = false;
```

Comment:
Are there any synchronization worries with this field? ie any reason to make it atomic/mutexed?

Reply:
No, it is only accessed from the loop thread.